src/model/storage/network.model.ts

```diff
@@ -0,0 +1,10 @@
+import { EthAddress } from '@ethersphere/bee-js/dist/types/utils/eth'
+
+export interface Network {
+  label: string
+  rpc: string
+  ensRegistry?: EthAddress
+  subdomainRegistrar?: EthAddress
+  publicResolver?: EthAddress
+  custom: boolean
```

Comment:
what is the purpose of this?

Reply:
That flag indicates whether a network object was created by the user or it is a predefined network. That's needed because we don't want to allow users to delete predefined networks.